roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -13,17 +14,20 @@
   when:
     cephx and
     groups[restapi_group_name] is defined
+  run_once: true
```

Comment:
Yeah, it's copied in the ceph-mon and ceph-osd roles.

On 13 Dec 2015, at 16:07, Madhav Puri wrote:

>> @@ -13,17 +14,20 @@
>> when:
>>   cephx and
>>   groups[restapi_group_name] is defined
>> +  run_once: true
>
> @erikh, not sure if this is a big one but I am wondering if this will 
> prevent the key from generated on all monitor hosts (except one). Do 
> we copy this key later to the monitor hosts? Otherwise we may not be 
> able to make REST API call on monitor hosts other than the first one.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/6/files#r47453237